app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadAdapter.kt

```diff
@@ -219,7 +219,7 @@ class ThreadAdapter(
         }
 
         if (item is MessageUi) {
-            (holder as MessageViewHolder).bindMail(item, position)
+            (holder as MessageViewHolder).bindMail(item, position, isReducible = items.size > 1)
```

Comment:
I know it's technically exactly the same, but we tend to prefer the Kotlin Extension here.
```suggestion
            (holder as MessageViewHolder).bindMail(item, position, isReducible = items.count() > 1)
```